client/v2/common/models/filtermodels.go

```diff
@@ -1,13 +1,5 @@
 package models
```

Comment:
These are the parameters to query objects. In the newly generated code they are declared inside the query file.